Cannot find module 'esprima-bin-wrapper.node.js'

Hi,

while trying to run Hunter I always get the same error message:

````
internal/modules/cjs/loader.js:638
    throw err;
    ^

Error: Cannot find module 'C:\C:\Users\<REDACTED>\Documents\Pharo\images\hunter\pharo-local\iceberg\dorellang\hunter\esprima-bin-wrapper\dist\esprima-bin-wrapper.node.js'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:636:15)
    at Function.Module._load (internal/modules/cjs/loader.js:562:25)
    at Function.Module.runMain (internal/modules/cjs/loader.js:831:12)
    at startup (internal/bootstrap/node.js:283:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:622:3)
````

**Notice the duplicate drive letter in the path: `C:\C:\`**

I'm running a 32bit Image of Pharo 7 on a 64bit Windows 10 Machine. I tried it with several of my own Javascript projects, as well as with [https://github.com/lodash/lodash](https://github.com/lodash/lodash).

I'm not sure if this really is a bug in Hunter or if I just messed up when tried to get it running.

--- a/src/BaselineOfHunter.package/BaselineOfHunter.class/instance/baseline..st
+++ b/src/BaselineOfHunter.package/BaselineOfHunter.class/instance/baseline..st
@@ -8,7 +8,7 @@ baseline: spec
 				spec repository: 'github://svenvc/NeoJSON:c368bbf475f60d564cd3fb708e99b92efea8ffd3/repository' ];
 
 			baseline: 'Roassal2' with: [
-				spec repository: 'github://ObjectProfile/Roassal2:c1da8614c951008ea501f126ef3ccf25d73580ac/src' ];
+				spec repository: 'github://ObjectProfile/Roassal2:984fee0822b33705bd094f35b314012a958b87ac/src' ];
 
 			baseline: 'Phanide' with: [
 				spec repository: 'github://ronsaldo/phanide:a38e384c9293bc1aa73fff0dd2396539a4500981/filetree' ];

--- a/src/Hunter-AST.package/HNEsprimaWrapper.class/class/baseDirectoryGuessIfNotFound..st
+++ b/src/Hunter-AST.package/HNEsprimaWrapper.class/class/baseDirectoryGuessIfNotFound..st
@@ -5,7 +5,7 @@ baseDirectoryGuessIfNotFound: aBlockClosure
 	| expectedPaths |
 	expectedPaths := {
 		FileLocator workingDirectory.
-		IceLibgitRepository repositoriesLocation / #dorellang / #hunter }.
+		FileLocator localDirectory / #iceberg / #dorellang / #hunter }.
 	expectedPaths 
 		detect: [ :each | (HNEsprimaWrapper esprimaWrapperPathRelativeTo: each) exists ]
 		ifFound: [ :each | ^ each ].

--- a/src/Hunter-AST.package/HNEsprimaWrapperRunStrategy.class/instance/esprimaWrapperArgumentsFor..st
+++ b/src/Hunter-AST.package/HNEsprimaWrapperRunStrategy.class/instance/esprimaWrapperArgumentsFor..st
@@ -1,8 +1,8 @@
 running
 esprimaWrapperArgumentsFor: aHunter
 	^ Array streamContents: [ :stream |
-		stream nextPut: HNEsprimaWrapper esprimaWrapperPath asPath pathString.
-	
+		stream nextPut: (self pathArgumentFor: HNEsprimaWrapper esprimaWrapperPath pathString).
+
 		aHunter scriptDirs ifNotEmpty: [ :aCollection |
 			stream nextPut: '-s'.
 			aCollection do: [:each | stream nextPut: (self pathArgumentFor: each) ].

